star-rx72n-firmware/lib/rx_usb_comm/src/rx_usb_comm.c

```
@@ -793,7 +934,17 @@ rx_usb_comm_receive(rx_usb_comm_handle_t* handle, rx_frame_t* frame, const uint3
 
     result = internal_receive_iteration(handle, frame, timeout_ms, &elapsed_ms, &err);
     if (result == k_receive_done) {
-      return k_rx_ok;
+      /* Dispatch control frames (PING, RESET) - consumed internally */
+      const usb_dispatch_result_t dispatch =
```

Clang (14.0.6)

[warning] variable 'dispatch' is not initialized

(cppcoreguidelines-init-variables)